liquidity-module/index.md

```diff
@@ -0,0 +1,175 @@
+
+```
+curl https://get.starport.network/starport@v0.16.0! | bash
+```
+
+You can also use Starport v0.16.0 in a [browser-based IDE](http://gitpod.io/#https://github.com/tendermint/starport/tree/v0.15.1). For more installation options, see [install Starport](http://gitpod.io/#https://github.com/tendermint/starport/tree/v0.16.0), but this tutorial assumes you are using a local Starport installation. See [install Starport](https://github.com/tendermint/starport/blob/develop/docs/intro/install.md).
+
+## Create the Blockchain
+
+Scaffold a new blockchain called `myblockchain`
+
+```bash
+starport app github.com/username/myblockchain
+cd myblockchain
+```
+
+## Add your token in the configuration.
+
+Open the `config.yml` file. 
+In the `accounts` parameter, add your username and new token:
+
+*Note: the minimum reserve coin amount for a pool in the liquidity module is 1,000,000, make sure to have enough tokens created*
+
+```yml
+accounts:
+	- name: username
+    coins: ["10000token", "50000000stake", "1000000000000mytoken"]
+```
+
+## Start your blockchain
+
+Start your blockchain with running the command in your terminal.
+
+```bash
+starport serve
+```
+
+You should see an output similar to this, with different account passphrases.
+
+```bash
+
+```
+
+## Configure the relayer
+
+The relayer is a software to connect two blockchains. Configure it with your endpoints so you can have connection between your blokchain and the testnet. After the connection is established, you will be able to send token from one blockchain to the other.
+
+*Note: If you already used the relayer before, you might want to remove existing relayer config: rm -r $HOME/.relayer/*
+
+*Note: If you already used the relayer before, delete all configuration that you used before rm -r $HOME/.starport/*
+
+Configure the relayer to create a connection between your local chain and the chain you want to connect to. In this example, the chain target will be the Gravity DEX testnet.
+
+```markdown
+starport relayer configure
+```
+
+For the local `source` chain, use the default values.
+For the testnet `target` chain, use the following values.
+
+
+Target RPC: [https://rpc.testnet.cosmos.network:443](https://rpc.testnet.cosmos.network/)
+
+Token Faucet: [https://faucet.testnet.cosmos.network:443](https://faucet.testnet.cosmos.network/)
+
+Target Gas Price (0.025uatom): 0.025stake
+
+
+Connect the chains
+
+```markdown
+starport relayer connect
+```
+
+## Get Token from the Faucet
+
+From the terminal output that `starport serve` created for you, use the `username` accounts address and claim token from the faucet
+
+```markdown
+curl -X POST -d '{"address": "cosmosxxxxx"}' https://faucet.testnet.cosmos.network
+```
+
+When you get a success message, you can check your balance.
+
+*See balance* [https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxxxx)
```

Comment:
Can we add a tip or something that cosmosxxx should be replaced by your address? Or is it very obvious?